content/en/docs/concepts/workloads/controllers/_index.md

```diff
@@ -1,5 +1,58 @@
+Pods would be a lot of effort. For example, if a `Pod` fails, you probably want to
+run a new Pod to replace it. Kubernetes can do that for you.
+
+You use Kubernetes API to create a workload
+{{< glossary_tooltip text="object" term_id="object" >}} that represent a higher level
+abstraction than a `Pod`, and then the Kubernetes
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}} automatically manages
+Pod objects on your behalf, based on the specification for the workload object you defined.
+
+The built-in APIs for managing workloads are:
+
+[`Deployment`](/docs/concepts/workloads/controllers/deployment/) (and, indirectly, [`ReplicaSet`](/docs/concepts/workloads/controllers/replicaset/)),
+the most common way to run an application on your cluster.
+A Deployment is a good fit for managing a stateless application workload on your cluster:
+_stateless_ means that any {{< glossary_tooltip term_id="Pod" >}} in the Deployment is interchangeable,
+and can be replaced if needed.
+(Deployments are a replacement for the legacy
+{{< glossary_tooltip text="ReplicationController" term_id="replication-controller" >}} API).
+
+A [`StatefulSet`](/docs/concepts/workloads/controllers/statefulset/) lets you
+manage one or more Pods – all running the same application code – that do track state somehow.
+For example, if your workload records data persistently, you can run a StatefulSet that
+associates each Pod with a [PersistentVolume](/docs/concepts/storage/persistent-volumes/).
+Your code, running in the Pods for that StatefulSet, can replicate data to
+other Pods in the same StatefulSet to improve overall resilience.
+
+A [`DaemonSet`](/docs/concepts/workloads/controllers/daemonset/) defines Pods that provide
+facilities that are local to a specific {{< glossary_tooltip text="node" term_id="node" >}};
+for example, a driver that lets containers on that node access a storage system. You use a DaemonSet
+when the driver, or other node-level service, has to run on the node where it's useful.
+Each Pod in a DaemonSet performs a job similar to a system daemon on a classic Unix / POSIX
+server.
+A DaemonSet might be fundamental to the operation of your cluster,
+such as a plugin to let that node access
+[cluster networking](/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-network-model),
+or it could provide less essential facilities that enhance the container platform you are running.
+You can run DaemonSets (and their pods) across every node in your cluster, or across just a subset (for example,
+only install the GPU accelerator driver on nodes that have a GPU installed).
+
+You can use a [`Job`](/docs/concepts/workloads/controllers/job/) and / or
+a [`CronJob`](/docs/concepts/workloads/controllers/cron-jobs/) to
+define tasks that run to completion and then stop. A Job represents a one-off task,
+whereas each CronJob repeats according to a schedule.
```

Comment:
```suggestion
You can use a [`Job`](/docs/concepts/workloads/controllers/job/)  to
define a task that runs to completion. You can use a
[`CronJob`](/docs/concepts/workloads/controllers/cron-jobs/) to run
the same Job multiple times according a schedule.
```

And since other workloads have examples, we can say:
For example, data analytics, CI/CD, HPC and AI/ML training applications.

Reply:
When this merges, you're very welcome to submit a follow-up PR with tweaks.

Reply:
Why not in this PR? It looks like this is new content.

Reply:
> Why not in this PR?

I'm afraid I'm short on capacity to incorporate all the improvements that are feasible to suggest.

Reply:
How's the revised text?